### PR TITLE
Binding to document.ready event implemented without jQuery (partial of #56)

### PR DESCRIPTION
**Pull request by helen-dikareva**
This PR'll close item "Get rid of jquery binding for 'ready' and 'load' events" from https://github.com/DevExpress/testcafe/issues/56 checklist
Wait for all qunit tests

### CODE REVIEW

File: src/client/core/utils/event.js [hunk 1]
```diff
@@ -31,3 +31,34 @@ export function unbind (el, event, handler, useCapture) {
     else
         nativeMethods.removeEventListener.call(el, event, handler, useCapture);
 }
+
+
+export function documentReady (callback) {
+    var isReady = false;
```
Comment (inikulin) [inline src/client/core/utils/event.js]: Maybe we'll use Promise for new API?

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
lgtm

**Comment by helen-dikareva** [comment]
Wait for ASP tests on farm

**Comment by helen-dikareva** [comment]
All tests done. @AlexanderMoskovkin please review

**Comment by inikulin** [comment]
ping @AlexanderMoskovkin

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**